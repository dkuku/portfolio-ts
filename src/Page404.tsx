import * as React from 'react'

const Page404: React.SFC<any> = () => {
    return (
      <div>
        Page not found
      </div>
    )
}

export default Page404;
