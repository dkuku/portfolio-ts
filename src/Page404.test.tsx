import { shallow } from 'enzyme';
import * as React from 'react';
import Page404 from './Page404';

describe('Page404', ()=>{
    it('should render without crashing and match snapshot', () => {
        const component = shallow(<Page404 />)
        expect(component).toMatchSnapshot();
    });
})
