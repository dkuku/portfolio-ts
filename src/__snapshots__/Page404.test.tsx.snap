// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 should render without crashing and match snapshot 1`] = `
<div>
  Page not found
</div>
`;
